packages/app-storage/src/Query.tsx

```diff
@@ -41,11 +43,51 @@ class Query extends React.PureComponent<Props, State> {
         value as Storage$Key$Value
       );
 
-      cache[id] = withStorageDiv(key, { params: values })(
-        (value: any) =>
-          valueToText(key.type, value),
-        { className: 'ui--output' }
-      );
+      // FIXME - Move into ui-app/src/Params/valueToAccountList for reuse
+      const renderAccounts = (addresses: Array<Uint8Array | string>) => {
+        return (
+          <table className='accounts'>
+            <thead>
+              <tr>
+                <th>Identitycon</th>
+                <th>Account ID</th>
+              </tr>
+            </thead>
+            <tbody>
+              {
+                addresses && addresses.length ?
+                  (
+                    addresses
+                      .map((address) => {
+                        return addressEncode(address as Uint8Array);
+                      })
+                      .map((accountId) => (
+                        <tr key={accountId}>
+                          <td><IdentityIcon size={24} value={accountId} /></td>
+                          <td>{accountId}</td>
+                        </tr>
+                      ))
+                  )
+                : null
+              }
+            </tbody>
+          </table>
+        );
+      };
+
+      if (key.type[0] === 'AccountId') {
```

Comment:
This is not correct, it only works for the specific instance and not for eg. Referendums where AccountId is not in that position. As per above, it needs to be on a single value (so each account, dfoesn't matter where gets rendered)

In some cases where we have tuples with AccountId first, we will now not display all info.

Reply:
Correct place is here https://github.com/polkadot-js/apps/blob/master/packages/ui-app/src/Params/valueToText.ts#L64